Asset/src/V1/Gapic/AssetServiceGapicClient.php

```diff
@@ -761,18 +761,24 @@ public function analyzeMove(
      *
      * @throws ApiException if the remote call fails
      */
-    public function batchGetAssetsHistory($parent, $contentType, $readTimeWindow, array $optionalArgs = [])
+    public function batchGetAssetsHistory($parent, array $optionalArgs = [])
```

Comment:
Well this doesn't seem right...

Looking at the proto files [here](https://github.com/googleapis/googleapis/commits/master/google/cloud/asset/v1/asset_service.proto), they haven't changed since August (and this library was updated in October).

I'm guessing there may have been something missed, either in AutoSynth or in some other component of generation?